### PR TITLE
Make minor configuration file edits (second try)

### PR DESCRIPTION
**Pull request by rhetoric101**
### Overview
This is a follow-up to this [PR](https://github.com/newrelic/newrelic-java-agent/pull/949/files) opened by @paperclypse.

Since I had difficulty pulling down the work from the fork of paperclypse, I asked for his file and created this new PR based on that. 

Based on that previous PR, it looked like all of the changes from paperclypse were accepted except moving the application name to the top. So, in this PR, I've moved the license key information to the top again.

I did notice that the original config file had a reference to code-level metrics and that this was removed in the version made by paperclypse. Alas, I don't see the Java agent listed as supporting code-level-metrics in CodeStream, so perhaps this should stay out of the config file (https://docs.newrelic.com/docs/codestream/how-use-codestream/performance-monitoring#code-level)?

Finally, I removed the new double dotted line introducing application name, since it's not required. Perhaps you want this restored?

### CODE REVIEW

**Comment by tbradellis** [comment]
Thank you @rhetoric101!  I will run through this tomorrow.

File: newrelic-agent/src/main/resources/newrelic.yml [hunk 1]
```diff
@@ -120,16 +120,9 @@ common: &default_settings
       # When true, the agent will decorate your application log files with linking metadata. The default is false.
       #enabled: false
 
-  # Adds integration with CodeStream, introducing Code-Level Metrics! Golden Signals visible in your
-  # IDE through New Relic CodeStream.
-  code_level_metrics:
-    # When true the agent will capture namespace and function information
-    # on spans to enable code level metrics in CodeStream.
-    enabled: false
-
```
Comment (tbradellis) [inline newrelic-agent/src/main/resources/newrelic.yml:129]: code_level_metrics and comments need to stay in. This is a new feature and new config that was recently merged and will release soon.

File: newrelic-agent/src/main/resources/newrelic.yml [hunk 2]
```diff
@@ -293,17 +286,18 @@ common: &default_settings
...
   # tracing, enable it with `cross_application_tracer.enabled = true` and
   # `distributed_tracing.enabled = false`
   cross_application_tracer:
 
-    # Set to true to enable cross application tracing.
-    # Default is false.
-    enabled: false
+    # Set to false to disable cross-application tracing.
+    # Default is true.
+    enabled: true
```
Comment (tbradellis) [inline newrelic-agent/src/main/resources/newrelic.yml:299]: This setting should be false. Distributed tracing replaced cross application tracing as the default.

**Review by tbradellis — CHANGES_REQUESTED** [review]

**Comment by jasonjkeller** [comment]
@rhetoric101 Do you intend to address the feedback on this PR and resolve the merge conflict? If we don't hear back from you in a few days we'll just close this PR so that our team JIRA story related to this isn't blocked indefinitely. Thanks!

**Comment by obenkenobi** [comment]
@rhetoric101 Just another heads up, do you intend to address the feedback and resolve the merge conflict in this PR? If we do not hear any feedback this coming Wednesday, i.e. September 28, a new PR will be made that will address the feedback and the merge conflicts. Thanks!

File: newrelic-agent/src/main/resources/newrelic.yml (hunk 2 above)
Comment (rhetoric101) [reply newrelic-agent/src/main/resources/newrelic.yml:299]: @tbradellis OK, I will change cross application tracing to "false," but this concerns me that this has been true while at the same time, distributed tracing was also true.

Comment (rhetoric101) [reply newrelic-agent/src/main/resources/newrelic.yml:299]: It's my understanding that cross application tracing has been false for a long time. I just pulled down the latest Java agent and the config file is showing "false" so I'm not sure how this got changed to "true."

**Comment by rhetoric101** [comment]
@jasonjkeller and @obenkenobi sorry I missed your comments! I'll take a look at these conflicts.

File: newrelic-agent/src/main/resources/newrelic.yml (hunk 2 above)
Comment (rhetoric101) [reply newrelic-agent/src/main/resources/newrelic.yml]: @tbradellis I resolved the conflict for cross application tracing by changing it to the value of "false."

**Comment by rhetoric101** [comment]
@jasonjkeller and @obenkenobi I resolved that conflict by changing the cross application tracing flag to "false." I'm not sure why that particular section was singled out. I looked at the original PR from @paperclypse and this was set as "true" there. Alas, it looks like this was not corrected in the the original [PR](https://github.com/newrelic/newrelic-java-agent/pull/949/files) and I pulled it into this one. 

This tells me that you should look closely at this file to confirm that the changes are all accurate. Sorry for the delay and confusion around this PR.

**Review by jasonjkeller — APPROVED** [review]

**Pull request merged**